streams/src/main/java/org/apache/kafka/streams/state/internals/ChangeLoggingSessionBytesStore.java

```diff
@@ -83,11 +83,11 @@ public void put(final Windowed<Bytes> sessionKey, final byte[] aggregate) {
 
     @Override
     public KeyValueIterator<Windowed<Bytes>, byte[]> fetch(final Bytes key) {
-        return findSessions(key, 0, Long.MAX_VALUE);
+        return bytesStore.fetch(key);
```

Comment:
In these cases, we're bypassing the underlying store for apparently no reason at all. It's not unreasonable to think that the underlying store could support more efficient versions of these operations.